src/scores-handler.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-cycle */
+import renderScores from './dom-handler';
+
+const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/LMXYZnWz9iXxoiT9U5dE/scores/';
+
+export const addScore = (name, score) => {
```

Comment:
* Kindly make use of async and await JavaScript features to consume the API as stated in the project [requirement.](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20async%20and%20await%20JavaScript%20features%20to%20consume%20the%20API.) it makes your code concise and clean it is also good for error handling [Read more](https://hackernoon.com/6-reasons-why-javascripts-async-await-blows-promises-away-tutorial-c7ec10518dd9)